.github/workflows/pr_test_peft_backend.yml

```diff
@@ -96,10 +96,11 @@ jobs:
         if [ "${{ matrix.lib-versions }}" == "main" ]; then
             python -m pip install -U peft@git+https://github.com/huggingface/peft.git --no-deps
             python -m uv pip install -U transformers@git+https://github.com/huggingface/transformers.git --no-deps
+            python -m uv pip install -U tokenizers@git+https://github.com/huggingface/tokenizers.git --no-deps
             pip uninstall accelerate -y && python -m uv pip install -U accelerate@git+https://github.com/huggingface/accelerate.git --no-deps
         else
             python -m uv pip install -U peft --no-deps
-            python -m uv pip install -U transformers accelerate --no-deps
+            python -m uv pip install -U transformers accelerate tokenizers --no-deps
```

Comment:
Does it break for the latest stable too?

Reply:
There are some failed runs on both `main` and `latest`, some of the more recent runs seem to be working though.

https://github.com/huggingface/diffusers/actions/workflows/pr_test_peft_backend.yml

Reply:
Should we try once more before the change then? Not a big deal.

Reply:
Yes this is ok now. Reviewing logs from failed runs it looks like `transformers` hadn't installed correctly https://github.com/huggingface/diffusers/actions/runs/12185630364/job/33992514505#step:5:19
https://github.com/huggingface/diffusers/actions/runs/12196730370/job/34024971182?pr=10125#step:5:19

Reply:
Alright then I am cool!

Reply:
I don't think it will cause issues if we merge this anyway but we can also just close.